src/blockstore-utils.js

```diff
@@ -13,11 +14,13 @@ const uint8ArrayToString = require('uint8arrays/to-string')
  * @returns {Key}
  */
 exports.cidToKey = cid => {
-  if (!CID.isCID(cid)) {
+  if (!(cid instanceof CID)) {
```

Comment:
I think the appropriate pattern to be using with the new `CID` is this:

```js
cid = CID.asCID(cid)
if (cid == null) { throw ... }
```

I don't think standard will let you `if ((cid = CID.asCID(cid)) != null)` but that'd be a neat shortcut.

It does an `instanceof` with a fast pass-through as the first check, but if that doesn't work it'll try a few other things and get us past the usual `instanceof` and version problems, looking for a _CID-like_, then return `null` if it can't make it work.

Reply:
```suggestion
  cid = CID.asCID(cid)
  if (cid == null) {
```